src/pages/kimnahee/Main/Main.scss

```
@@ -0,0 +1,370 @@
+@import '../Login/Login.scss';
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
+
+$MainPrimary-border: 2px solid #efefef;
+
+.nav {
+  border-bottom: 1px solid rgb(200, 200, 200);
+  background-color: #ffffff;
+
+  .instaIcon {
+    width: 50px;
+    border-right: 1px solid gray;
+    margin: 20px 20px 20px 30px;
+    padding-top: 5px;
+  }
+  .instaLogo {
+    font-family: $primary-font;
+    font-size: 35px;
+    margin-top: 15px;
+  }
+  .navBox {
+    @include flexSort(space-between);
+    flex-direction: row;
+    margin: 0 100px;
+    align-items: center;
+  }
+  .logoIcon {
+    display: flex;
+  }
+
+  .search {
+    width: 250px;
+    height: 30px;
+    background-color: rgb(240, 240, 240);
+    border: 1px solid rgb(200, 200, 200);
+    text-align: center;
+    margin: 0;
+  }
+
+  .search::-webkit-input-placeholder {
+    background-image: url(https://cdn1.iconfinder.com/data/icons/hawcons/32/698627-icon-111-search-256.png);
+    background-size: contain;
+    background-position: 35% center;
+    background-repeat: no-repeat;
+    text-align: center;
+    text-indent: 0;
+  }
```

```suggestion
  .search {
    width: 250px;
    height: 30px;
    background-color: rgb(240, 240, 240);
    border: 1px solid rgb(200, 200, 200);
    text-align: center;
    margin: 0;
    
    &::-webkit-input-placeholder {
      background-image: url(https://cdn1.iconfinder.com/data/icons/hawcons/32/698627-icon-111-search-256.png);
      background-size: contain;
      background-position: 35% center;
      background-repeat: no-repeat;
      text-align: center;
      text-indent: 0;
    }
  }
```

이런 식으로 `&`를 이용하면 어떨까요?